pkg/virt-controller/watch/migration/migration.go

```diff
@@ -1799,6 +1825,35 @@ func (c *Controller) deleteResourceQuota(obj interface{}) {
 	return
 }
 
+func (c *Controller) updateKubeVirt(org, cur interface{}) {
+	curKubevirt := cur.(*virtv1.KubeVirt)
+	orgKubevirt := org.(*virtv1.KubeVirt)
+
+	curSyncAddress := getSynchronizationAddress(curKubevirt)
+	orgSyncAddress := getSynchronizationAddress(orgKubevirt)
+	if curSyncAddress != orgSyncAddress && curSyncAddress != "" {
+		// sync address was updated, update all active migrations
+		for _, obj := range c.migrationIndexer.List() {
+			migration, ok := obj.(*virtv1.VirtualMachineInstanceMigration)
+			if !ok {
+				log.Log.Errorf("found unknown object in kubevirt store %v", obj)
```

Comment:
This is not the KV store, it's the VMIM store?

Reply:
Copy and paste and forgot to change the message